packages/dnsmap/PKGBUILD

```diff
@@ -2,30 +2,43 @@
 # See COPYING for license details.
 
 pkgname=dnsmap
-pkgver=0.30
-pkgrel=4
-groups=('blackarch' 'blackarch-fingerprint')
+pkgver=0.36.r6.g96b17ad
+pkgrel=1
 pkgdesc='Passive DNS network mapper.'
 arch=('x86_64' 'aarch64')
-url='http://dnsmap.googlecode.com'
-license=('GPL')
-makedepends=('make')
+groups=('blackarch' 'blackarch-fingerprint')
+url='https://github.com/resurrecting-open-source-projects/dnsmap'
+license=('GPL3')
+makedepends=('git' 'make')
 install=dnsmap.install
-source=("http://dnsmap.googlecode.com/files/$pkgname-$pkgver.tar.gz")
-sha512sums=('a9a8a17102825510d16c1f8af33ca74407c18c70')
+source=("git+https://github.com/resurrecting-open-source-projects/$pkgname.git")
```

Comment:
The original (abandoned) seems to be hosted at https://github.com/makefu/dnsmap.
https://github.com/resurrecting-open-source-projects/dnsmap seems to be a continuation / revival so it's more up to date, but the README clearly states it was moved to a new repository at https://github.com/pagvac/dnsmap.

So either we use the old https://github.com/makefu/dnsmap for archive purpose, or https://github.com/pagvac/dnsmap for the revival, but not https://github.com/resurrecting-open-source-projects/dnsmap.

Reply:
Thanks. The revived project has been reworked to Python.